workflow/comp_pred_ridge_and_LSTM.ipynb

```diff
@@ -0,0 +1,2196 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ground_truth = test_y_torch.squeeze().numpy()\n",
+    "\n",
+    "print(f\"The MSE loss is {hist_test[0]:.3f}\")\n",
```

Comment:
Can you add the MSE loss for the mean ground truth here as well? Should be simple to calculate.

Reply:
Yes, I will calculate MSE loss for the mean of training data (not ground truth, just to be fair).